modules/oadp-release-notes-1-1-4.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/oadp-release-notes.adoc
+
+:_content-type: REFERENCE
+[id="migration-oadp-release-notes-1-1-4_{context}"]
+= OADP 1.1.4 release notes
+
+The OADP 1.1.4 release notes lists any new features, resolve issues and bugs, and known issues.
+
+[id="new-features1.1.4_{context}"]
+== New features
+
+This version of OADP is a service release. No new features are added to this version.
+
+[id="resolved-issues1.1.4_{context}"]
+== Fixed bugs
+
+The following bugs have been fixed in this release:
+
+* link:https://issues.redhat.com/browse/OADP-1557[OADP-1557]
+* link:https://issues.redhat.com/browse/OADP-1822[OADP-1822]
+* link:https://issues.redhat.com/browse/OADP-1511[OADP-1511]
+* link:https://issues.redhat.com/browse/OADP-1642[OADP-1642]
+* link:https://issues.redhat.com/browse/OADP-1398[OADP-1398]
+* link:https://issues.redhat.com/browse/OADP-1267[OADP-1267]
+* link:https://issues.redhat.com/browse/OADP-1390[OADP-1390]
+* link:https://issues.redhat.com/browse/OADP-1650[OADP-1650]
+* link:https://issues.redhat.com/browse/OADP-1487[OADP-1487]
+
+
+[id="known-issues1.1.4_{context}"]
+== Known issues
+
+This release has the following known issues:
+
+* Issues restoring an OADP backup, with the application unable to access data. UID / GID range may have changed on the cluster where the application has been restored, with the result that OADP does not backup and restore OpenShift UID/GID range metadata. If the backed application requires a specific UUID, ensure the range is available when restored. Workaround: Allowing OADP to create the namespace while restoring may avoid the issue.
+
+* It is possible that the restore will fail if ArgoCD is used during the restore. This failure could be caused by a label used by ArgoCD `app.kubernetes.io/instance`. This label is used to identify which resources ArgoCD needs to manage, which can create conflict with OADP managing resources on restore. Workaround: setting `.spec.resourceTrackingMethod` on the ArgoCD YAML to `annotation+label` or `annotation`. If the issue continues to persist, then disable ArgoCD before beginning the restore, and enable when the restore has completed.
+
+
```

Comment:
Suggestion:  A restore might fail if ArgoCd is used during the restore, due to  a label used by ArgoCD, `app.kubernetes.io/instance`. This label identifies which resources ArgoCD needs to manage, which can create a conflict with OADP's procedure for managing resources on restore. 

Workaround:  Set `.spec.resourceTrackingMethod` on the ArgoCD YAML to `annotation+label` or `annotation`. If the issue continues to persist, then disable ArgoCD before beginning the restore, and enable it again when the restore has completed.